src/components/braintree/BraintreePayPalButtons.tsx

```diff
@@ -47,6 +48,7 @@ export const BraintreePayPalButtons: FC<
                     .then((clientInstance) => {
                         return braintree.paypalCheckout.create({
                             client: clientInstance,
+                            ...{ merchantAccountId },
```

Comment:
Are we able to do this instead?
```suggestion
                            merchantAccountId,
```